util/test/checkChplInstall

```diff
@@ -96,6 +96,30 @@ else
     CHPL_DIR_REMOVE=0
 fi
 
+# Tmp directory for compile output
+TMP_TEST_DIR="$(mktemp -d ${CHPL_DIR}/chapel-test-XXXXX)"
+log_info "Temporary test job directory: ${TMP_TEST_DIR}"
+
+# Cleanup the tmp directory whenever exit is invoked
+function cleanup_tmp_dir()
+{
+    local i
+    log_debug "Removing ${TMP_TEST_DIR}"
+    for i in 1 2 3 4 5
+    do
+        rm -rf ${TMP_TEST_DIR} && break
+        # sometimes rm needs to wait for the TEST_JOB exe file to close
```

Comment:
Is this part of the reason that tmp dirs were sometimes left around?
Do you have any evidence that this loop with sleeps solves the issue?

I don't have a problem keeping this loop here but I think it should echo something out to the user to indicate what is happening, in case it causes trouble later.